log.py

```diff
@@ -12,6 +12,8 @@
 
 from . import conf, world
 
+__all__ = ['log', 'PyLinkChannelLogger']
```

Comment:
I would leave out PyLinkChannelLogger and import it explicitly in classes.py - its usage is pretty specific